ansible_base/resource_registry/utils/sync_to_resource_server.py

```diff
@@ -67,8 +67,9 @@ def sync_to_resource_server(instance, action, ansible_id=None):
     else:
         logger.error("No user found, syncing to resource server with jwt_user_id=None")
 
+    service_path = getattr(settings, "RESOURCE_SERVICE_PATH", "/api/gateway/v1/service-index/")
```

Comment:
I'd argue that this should be set as the default value in `ansible_base/lib/dynamic_config/settings_logic.py`, as opposed to here.

This is because the setting should be reused in this line, instead of having it hard-coded.

https://github.com/ansible/django-ansible-base/blob/fa97197950ee806659d61e7591183fa391468d00/ansible_base/resource_registry/tasks/sync.py#L68

Right?

Reply:
I just removed that item from params on this line, as we set it below (as long as service_path is not None or empty)

hopefully that is the correct logic